src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-size;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #{$main-text-color};
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    color: #{$second-text-color};
+    font-weight: 400;
+
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__rating-review {
+    font-size: 10px;
+    color: #{$main-text-color};
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+  }
+
+  &__price-name {
+    color: #{$second-text-color};
+    font-weight: 400;
+  }
+
+  &__price-price {
+    font-size: 16px;
+    color: #{$main-text-color};
+    font-weight: 700;
+  }
+
+  &__buy {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    font-weight: 700;
+
+    text-decoration: none;
+    background: #{$button-color};
+    border: 1px solid #{$button-color};
+    border-radius: 5px;
+    text-transform: uppercase;
+
+    &:hover {
+      color: #{$button-color};
```

Comment:
don't need tipe #{} when connect variables in one file through import, use only $ (everywhere)